contracts/universalSchemes/ContributionReward.sol

```diff
@@ -45,7 +45,7 @@ contract ContributionReward is UniversalScheme {
     }
 
     // A mapping from the organization (Avatar) address to the saved data of the organization:
-    mapping(address=>mapping(bytes32=>ContributionProposal)) public organizationsProposals;
+    mapping(address=>mapping(bytes32=>ContributionProposal)) public organizationProposals;
```

Comment:
organizationProposals -> organizationsProposals 
please revert it back

Reply:
I'm not sure where this diff is coming from, it is name `organizationProposals` [in the code](https://github.com/daostack/arc/blob/eb7b6965105d32dec990fb6409f4841d86944ad7/contracts/universalSchemes/ContributionReward.sol#L48).

Reply:
it should be name organizationsProposals